wordpress-mobile/AztecEditor-Android
# Feature/rtl toolbar

This PR fixes toolbar in RTL direction.

[![https://gyazo.com/a388afe138a6960ebc43b21e1a945cba](https://i.gyazo.com/a388afe138a6960ebc43b21e1a945cba.gif)](https://gyazo.com/a388afe138a6960ebc43b21e1a945cba)

To test:

1. Change locale to RTL language or toggle "Force RTL layout direction" in Dev options.
2. Check that basic and advanced toolbars work in RTL mode.

Note: Toolbar is using `HorizontalScrollView` which did not get  RTL support until API 19. On API 17 and 18 LTR layout will be forced.
